test.rb

```
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require './utils'
+
+def fetch_albums
+  File.readlines('discography.txt').map(&:chomp)
+end
+
+def get_albums_by_decade(albums)
+  result = {}
+  albums.each do |album|
+    year = album[0..2].to_i * 10
+    next if album.nil?
+
+    if result[year].nil?
+      result[year] = [album]
+    else
+      result[year] << album
+    end
+  end
+  result
+end
+
+def create_list_and_cards(decade, albums, board_id)
+  list = create_list(decade, board_id)
+  if list.nil?
+    puts "List #{decade} couldn't be created"
+    return
```

This seems like a big deal that should raise an exception